plugins/kubernetes-backend/src/types/types.ts

```diff
@@ -14,16 +14,22 @@
  * limitations under the License.
  */
 
+import { Entity } from '@backstage/catalog-model';
 import { Logger } from 'winston';
 import type { JsonObject } from '@backstage/types';
 import type {
   FetchResponse,
   KubernetesFetchError,
+  KubernetesRequestAuth,
   KubernetesRequestBody,
   ObjectsByEntityResponse,
 } from '@backstage/plugin-kubernetes-common';
 import { PodStatus } from '@kubernetes/client-node/dist/top';
 
+/**
+ *
+ * @alpha
```

Comment:
Using `@alpha` here is perhaps a bit confusing. We've added support for separating `@alpha` APIs into a separate package entrypoint, but it needs to be enabled in the package `build` command. I don't think it's quite what you're going for though right? Probably want to just use `@public` for these instead.

Reply:
I think we want to set these to these all to alpha as the interface is still evolving and we're not ready to commit to it as is right now

Reply:
Yep makes sense, just clarifying that it's a pretty weak signal and nothing people will really pay attention to in practice